README.md

```diff
@@ -16,27 +17,59 @@ For installation, configuration, and details about using the API, check out our
 
 For descriptions of terminology used in APM, take a look at the [official documentation](https://docs.datadoghq.com/tracing/visualization/).
 
+
+## Version Release Lines and Maintenance
+
+We currently maintain two release lines, namely `v2` and `v3`.
+Features and bug fixes that are merged are then released to both the `v2` and `v3` release line.
+
+For any new projects it is recommended to use the `v3` release line.
+Currently, `v3` can be installed by running the following commands:
+
+```sh
+npm install dd-trace
+yarn add dd-trace
+```
+
+However, existing projects that already use the `v2` release line, or projects that need to support Node.js v12 and older, may use the `v2` release line.
+This can be installed by specifying the version when running an install command:
+
+```sh
+npm install dd-trace@2
+yarn add dd-trace@2
+```
+
+Any backwards-breaking functionality that is introduced into the library will result in an increase of the major version of the library.
+Such releases are kept to a minimum to reduce the pain of upgrading the library.
+Generally, newer versions of the library will be supported longer than older versions.
```

Comment:
TODO: Need to list the exact policy on LTS and support stuff.

TODO: Should we put in a small chart? Kinda like [Node.js LTS](https://github.com/nodejs/release#release-schedule) or what [.NET](https://github.com/DataDog/dd-trace-dotnet#downloads) sort of does?